output/csv/consts.go

```diff
@@ -0,0 +1,20 @@
+// Package csv custom enum types used in config
```

Comment:
This is comment for the whole *package*, but you are basically documenting the file. 

You likely did this as you were told by the linter, but it will be better if you add as separate `doc.go` file and just add some very basic doc such has `Package csv implements an output writing metrics in csv format`

Reply:
Thanks for the suggestion, done.